src/site/stages/build/plugins/add-id-to-subheadings.js

```diff
@@ -61,15 +61,10 @@ function generateHeadingIds() {
             !isInAccordion
           ) {
             tableOfContents.append(
-              `<li class="vads-u-margin-bottom--2">
-                  <a href="#${heading.attr(
-                    'id',
-                  )}" onClick="recordEvent({ event: 'nav-jumplink-click' });"
-              class="vads-u-display--flex vads-u-text-decoration--none">
-                    <i class="fas fa-arrow-down va-c-font-size--xs vads-u-margin-top--1 vads-u-margin-right--1"></i>
-                    ${heading.text()}
-                  </a>
-                </li>`,
+              `<li class="vads-u-margin-bottom--2"><a href="#${heading.attr(
+                'id',
+              )}" onClick="recordEvent({ event: 'nav-jumplink-click' });"
+              class="vads-u-display--flex vads-u-text-decoration--none"><i class="fas fa-arrow-down va-c-font-size--xs vads-u-margin-top--1 vads-u-margin-right--1"></i>${heading.text()}</a></li>`,
```

Comment:
If permitted by the linter, can we shuffle the indentation around so this so that it looks more like the previous version? It's hard to tell what actually changed and the I think the previous indentation is more readable.

Reply:
Done